lib/restClient.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable camelcase */
+'use strict';
+
+const {promisify} = require('util');
+const request = promisify(require('request'));
+
+module.exports = function (emitter, cfg) {
```

Comment:
maybe it's better to use ES2015 class syntax here since you're instantiating restClient with `new`?